Documentation/ApiOverview/SiteHandling/SiteSets.rst

```diff
@@ -0,0 +1,345 @@
+..  include:: /Includes.rst.txt
+..  index:: Site handling; Site sets
+..  _site-sets:
+
+=========
+Site sets
+=========
+
+Site sets ship parts of the site configuration as composable pieces. They are
+intended to deliver settings, TypoScript, TSconfig and reference enabled content
+blocks for the scope of a site.
+
+Extensions can provide multiple sets in order to ship presets for different
+sites or subsets (think of frameworks) where selected features are exposed
+as a subset (example: `typo3/seo-xml-sitemap`).
+
+..  contents:: Table of content
+    :local:
+
+..  _site-sets-definition:
+
+Site set definition
+===================
+
+A set is defined in an extension's sub folder in :path:`Configuration/Sets/`, for
+example :file:`EXT:my_extension/Configuration/Sets/MySet/config.yaml`.
+
+The folder name in :path:`Configuration/Sets/` is arbitrary, significant
+is the `name` defined in :file:`config.yaml`. The `name` uses a `vendor/name`
+scheme by convention, and *should* use the same vendor as the containing
+extension. It may differ if needed for compatibility reasons (for example when
+sets are moved to other extensions). If an extension provides exactly one set
+that should have the same `name` as defined in :file:`composer.json`.
+
+The :file:`config.yaml` for a set that is composed of three subsets looks as
+follows:
+
+..  code-block:: yaml
+    :caption: EXT:my_extension/Configuration/Sets/MySet/config.yaml
+
+    name: my-vendor/my-set
+    label: My Set
+
+    # Load TypoScript, TSconfig and settings from dependencies
+    dependencies:
+      - some-namespace/slider
+      - other-namespace/fancy-carousel
+
+..  _site-sets-usage:
+
+Using a site set as dependency in a site
+========================================
+
+Sets are applied to sites via `dependencies` array in site configuration:
+
+..  code-block:: yaml
+    :caption: config/sites/my-site/config.yaml
+
+    base: 'http://example.com/'
+    rootPageId: 1
+    dependencies:
+      - my-vendor/my-set
+
+Site sets can also be added to a site via the backend module
+:guilabel:`Site Management > Sites`.
+
+..  _site-sets-settings-definition:
+
+Settings definitions
+====================
+
+Sets can define settings definitions which contain more metadata than just a
+value: They contain UI-relevant options like `label`, `description`, `category`
+and `tags` and types like `int`, `bool`, `string`, `stringlist`, `text` or
+`color`. These definitions are placed in :file:`settings.definitions.yaml`
```

Comment:
Enhance this paragraph with something like "This YAML file resembles what extensions can provide via :file:`ext_emconf.txt` as additional configuration"? I think that could help people understand what this is tryint to enhance/replace?

Reply:
I don't really get what you are trying to explain here. To my knowledge there was never a ext_emconf.txt and the ext_emconf.php contains completely different information. Feel free to make a follow up, though